Disable `Avatar` & `AvatarGroup` `variant` (#1422)
iTwin/stratakit: apps/website/src/content/docs/components/mui/Avatar.md

- Restyled using StrataKit's visual language.
- The default size has been decreased.
- Requires a nested `<abbr>` for initials.

> - The `variant` prop defaults to `"circular"`, and does not support `"rounded"` or `"square"`.

Reviewer: Wording implies that the default was changed, but "circular" is the default w/o our modifications. (this could be reworded even more clearly if the prop is removed all together)

```suggestion
- The `"rounded"` and `"square"` variants have been removed. 
```

Author: Valid point, suggestion committed.
